src/app/app-routing.module.ts

```diff
@@ -41,7 +40,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { useHash: true })],
```

Comment:
I guess, it's adding # in all the urls, can we confirm this, this should be for this generate link url only and not for others

Reply:
No need, will remove it